src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets

```diff
@@ -104,8 +104,7 @@ Copyright (c) .NET Foundation. All rights reserved.
                       '$(_TargetFrameworkVersionWithoutV)' >= '3.0' And '$(TargetFrameworkIdentifier)' == '.NETCoreApp' "
           DependsOnTargets="_RunILLink">
 
-    <NETSdkInformation ResourceName="ILLink_Info" />
-
+    <NETSdkInformation Condition="'$(_ILLinkExitCode)' == '' Or '$(_ILLinkExitCode)' == '0'" ResourceName="ILLinkOptimizedAssemblies" />
```

Comment:
Personally I'm not a fan of writing this message. Honestly we shouldn't have any message really (Even before ILLink), but it's so slow that we kind of have to. But I don't see any reason to put one after it's done. There's little value to it. I would only keep the one before.

That also means moving the http link to the "start" message.